libs/sdk-ui-kit/styles/scss/stylingPicker.scss

```diff
@@ -0,0 +1,189 @@
+@import "./variables";
+
+.gd-styling-picker {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+    width: 100%;
+    padding: 20px;
+    background-color: $gd-color-white;
+    border: 1px solid $is-focused-background;
+
+    .gd-styling-picker-title {
+        display: flex;
+        padding-bottom: 10px;
+        line-height: 23px;
+        cursor: default;
+
+        h3 {
```

Comment:
I'm not fan of these nested css selectors (+ vague selectors as h3), imo, best is to keep everything as flat as possible (once you start moving it later, or making css overrides, it's a pain)

Reply:
I flattened the styles and removed the h3